HaulageApplication/Haulage/AdminPages/ManageCustomerPage.xaml.cs

```diff
@@ -0,0 +1,34 @@
+using Haulage.DatabaseExecutionServices;
+
+namespace Haulage.AdminPages;
+
+public partial class ManageCustomerPage : ContentPage
+{
+	public ManageCustomerPage()
+	{
+		InitializeComponent();
+        AddEmployee.Clicked += async (sender, args) => { await Navigation.PushAsync(new AddEmployee()); };
```

Comment:
There is nothing wrong with this way of adding the clicked event handler.

But just wondering whether, since the other handlers are added in an alternative way (handler associated with event in the xaml file, and the handler implemented as a method in the xaml.cs file), for consistency all handlers should be added in the same way? (i.e. xaml/xaml.cs approach)

Reply:
have changed this to be in line with the previous pages